integration_tests/beforeall-disabled/skip.test.js

```diff
@@ -1,11 +1,13 @@
-describe("abc", function() {
-  it("runs abc", function() {});
+/* globals fail */
```

Comment:
@cpojer yet another jasmine global... Can we add it to the `jest` global?

Reply:
I'd rather just kill this one and provide a better API tbh.